cmd/helm/root.go

```diff
@@ -68,10 +70,17 @@ func newRootCmd(actionConfig *action.Configuration, out io.Writer, args []string
 
 	// Add the registry client based on settings
 	// TODO: Move this elsewhere (first, settings.Init() must move)
+	// TODO: handle errors
```

Comment:
Would you mind refactoring this to capture these errors? Perhaps this is a good time to call `panic()`. @adamreese any thoughts how we could better handle client initialization like we do in `newActionConfig`?

Reply:
Im happy to use panic for now. The method signature just did allow for return of error object

Reply:
I've added `panic`s in latest commit, with a TODO to return the error when possible